_posts/0000-01-02-bertab63.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Slides are fun in the playground.  I enjoy sliding on the slide and swinging on the swingset.